web/pages/slow.svelte

```diff
@@ -41,12 +43,29 @@
       </ul>
       {#if $slowPosts.isLoading}
         <Loader />
-      {:else if $nextSlowSince}
+      {/if}
+      {#if $totalSlowPages > 1}
+        <PaginationBar
+          currentPage={$currentSlowPage}
+          label={`${$totalSlowPosts} ${$t.posts}`}
+          totalPages={$totalSlowPages}
+          on:click={e => {
+            openRoute({
+              params: {
+                currentPage: e.detail,
+                feed: $slowFeed
+              },
+              route: 'slow'
+            })
+          }}
+        />
+      {/if}
+      {#if $currentSlowPage < $totalSlowPages}
```

Comment:
The next-page button you mean?

Reply:
Yes. For me, it looks like this is part of the same control, since they both do the same job.

If we add pagination somewhere else, we will add there Next page button too.